### ISSUE TITLE
ERROR: testPrintRawUtf8String (google.protobuf.internal.text_format_test.TextFormatTest)

### ISSUE THREAD

**Issue by juannorris** [machinalis/protobuf-python3#54]
I've been checking python2.6 tests after every merge. Found this error, I think after #51,  #39 or #45.
# 
## ERROR: testPrintRawUtf8String (google.protobuf.internal.text_format_test.TextFormatTest)

Traceback (most recent call last):
  File "/home/jnorris/projects/protobuf/python/google/protobuf/internal/text_format_test.py", line 212, in testPrintRawUtf8String
    self.CompareToGoldenText(text, b'repeated_string: "\303\274\352\234\237"\n')
  File "/home/jnorris/projects/protobuf/python/google/protobuf/internal/text_format_test.py", line 61, in CompareToGoldenText
    self.CompareToGoldenLines(text, golden_text.splitlines(1))
  File "/home/jnorris/projects/protobuf/python/google/protobuf/internal/text_format_test.py", line 67, in CompareToGoldenLines
    ''.join(difflib.ndiff(golden_lines, actual_lines)))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 20: ordinal not in range(128)

**Comment by ramiro** [comment]
It is because the fix for #39.

**Comment by dmoisset** [comment]
According to git bisect, this is the offending commit:

```
c816517ebe36ba2c2812c5090be0b4db7b153741 is the first bad commit
commit c816517ebe36ba2c2812c5090be0b4db7b153741
Author: Ramiro Morales <rmorales@machinalis.com>
Date:   Tue Jul 31 18:10:18 2012 -0300

    Fix building of error message in call to assertEqual() that involves a call to difflib.ndiff().

:040000 040000 4aa6cff6a24dd6572dfa9dbdfd452d3c5e2198fe 7c419260bd6b7f470671edccc9bc260457ca9812 M  python
```

Ramiro, please take a look to see if you can find a fix for #39 that doesn't break 2.6 compatibility

**Comment by juannorris** [comment]
Fixed in a71248937f88c99312ea9316684d1820f0cdf42a.
